### DISCUSSION TITLE
Baronfig notebooks

### DISCUSSION THREAD

**Discussion by ryanprior** [elementary/store#155] [Product Ideas]
Baronfig does excellent branded gear: https://baronfig.com/business/branded

I like to buy their notebooks & would buy a whole set with elementary branding =D

**Comment by cassidyjames** [comment]
Hm, interesting! The store is currently backed by [Printful](https://printful.com) because we can use their API to print and ship on demand rather than carrying stock and manually processing orders and shipping out from one of our distributed homes. I don't think that's going to change any time soon, so I don't think we'd be able to offer this.
